recipe/isl-strdup-win.patch

```diff
@@ -0,0 +1,336 @@
+diff --git a/include/isl/ctx.h b/include/isl/ctx.h
+index ecd9272e..3fcba36f 100644
+--- a/include/isl/ctx.h
++++ b/include/isl/ctx.h
+@@ -233,7 +233,7 @@ isl_stat prefix ## _set_ ## field(isl_ctx *ctx, const char *val)	\
+ 	if (!val)							\
+ 		return isl_stat_error;					\
+ 	free(options->field);						\
+-	options->field = strdup(val);					\
++	options->field = __strdup(val);					\
```

Comment:
There are two underscores here, is this intentional?

Reply:
Thanks, good catch! 94b7af0